drivers/scheduler/scheduler.go

```diff
@@ -225,7 +225,13 @@ type Driver interface {
 	DeleteSnapShot(ctx *Context, snapshotName, snapshotNameSpace string) error
 
 	// GetSnapshotsInNameSpace get the snapshots list for the namespace
-	GetSnapshotsInNameSpace(ctx *Context, snapshotNameSpace string) (*snapv1.VolumeSnapshotList, error)
+	GetSnapshotsInNameSpace(ctx *Context, snapshotNameSpace string) (*volsnapv1.VolumeSnapshotList, error)
+
+	//DeleteCsiSnapshotsFromNamespace deletes the all snapshots from a namespace
+	DeleteCsiSnapshotsFromNamespace(ctx *Context, namespace string) error
+
+	//IsCsiSnapshotExists checks if a snapshot exists in the particular namespace
+	IsCsiSnapshotExists(ctx *Context, snapshotName string, namespace string) (bool, error)
```

Comment:
We can get namespace using `ctx.App.NameSpace`. How about we remove namespace argument to reduce redundancy?

Reply:
May be we can ignore this for now as it is consistent with GetSnapshotsInNameSpace. We can take this up in future.

Reply:
we need this to validate each individual snapshots